configs/default/NERC-NEUTRONRCH7BT.config

```diff
@@ -0,0 +1,192 @@
+resource ADC_BATT 1 C00
+resource ADC_RSSI 1 C05
+resource ADC_CURR 1 C01
+resource ADC_EXT 1 C04
+resource SDIO_CK 1 C12
+resource SDIO_CMD 1 D02
+resource SDIO_D0 1 C08
+resource SDIO_D1 1 C09
+resource SDIO_D2 1 C10
+resource SDIO_D3 1 C11
+resource PINIO 1 C03
+resource PINIO 2 D10
+resource PINIO 3 D11
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 B02
+resource GYRO_EXTI 2 E15
+resource GYRO_CS 1 C15
+resource GYRO_CS 2 E11
+resource USB_DETECT 1 E02
+
+# timer
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer A00 AF2
+# pin A00: TIM5 CH1 (AF2)
+timer A01 AF2
+# pin A01: TIM5 CH2 (AF2)
+timer A02 AF2
+# pin A02: TIM5 CH3 (AF2)
+timer A03 AF2
+# pin A03: TIM5 CH4 (AF2)
+timer D12 AF2
+# pin D12: TIM4 CH1 (AF2)
+timer D13 AF2
+# pin D13: TIM4 CH2 (AF2)
+timer D14 AF2
+# pin D14: TIM4 CH3 (AF2)
+timer D15 AF2
+# pin D15: TIM4 CH4 (AF2)
+timer E05 AF4
+# pin E05: TIM15 CH1 (AF4)
+timer E06 AF4
+# pin E06: TIM15 CH2 (AF4)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer A15 AF1
+# pin A15: TIM2 CH1 (AF1)
+timer C07 AF3
+# pin C07: TIM8 CH2 (AF3)
+timer C06 AF3
+# pin C06: TIM8 CH1 (AF3)
+timer B08 AF1
+# pin B08: TIM16 CH1 (AF1)
+timer B09 AF1
+# pin B09: TIM17 CH1 (AF1)
+
+# dma
+dma ADC 1 8
+# ADC 1: DMA2 Stream 0 Request 9
+dma ADC 3 9
+# ADC 3: DMA2 Stream 1 Request 115
+dma TIMUP 1 0
+# TIMUP 1: DMA1 Stream 0 Request 15
+dma TIMUP 2 0
+# TIMUP 2: DMA1 Stream 0 Request 22
+dma TIMUP 3 2
+# TIMUP 3: DMA1 Stream 2 Request 27
+dma TIMUP 4 1
+# TIMUP 4: DMA1 Stream 1 Request 32
+dma TIMUP 5 0
+# TIMUP 5: DMA1 Stream 0 Request 59
+dma TIMUP 8 0
+# TIMUP 8: DMA1 Stream 0 Request 51
+dma pin B00 0
+# pin B00: DMA1 Stream 0 Request 25
+dma pin B01 1
+# pin B01: DMA1 Stream 1 Request 26
+dma pin A00 2
+# pin A00: DMA1 Stream 2 Request 55
+dma pin A01 3
+# pin A01: DMA1 Stream 3 Request 56
+dma pin A02 4
+# pin A02: DMA1 Stream 4 Request 57
+dma pin A03 5
+# pin A03: DMA1 Stream 5 Request 58
+dma pin D12 6
+# pin D12: DMA1 Stream 6 Request 29
+dma pin D13 7
+# pin D13: DMA1 Stream 7 Request 30
+dma pin D14 12
+# pin D14: DMA2 Stream 4 Request 31
+dma pin E05 0
+# pin E05: DMA1 Stream 0 Request 105
+dma pin A08 14
+# pin A08: DMA2 Stream 6 Request 11
+dma pin A15 0
+# pin A15: DMA1 Stream 0 Request 18
+dma pin C07 0
+# pin C07: DMA1 Stream 0 Request 48
+dma pin C06 0
+# pin C06: DMA1 Stream 0 Request 47
+dma pin B08 0
+# pin B08: DMA1 Stream 0 Request 109
+dma pin B09 0
+# pin B09: DMA1 Stream 0 Request 111
+
+# feature
+feature RX_SERIAL
+feature TELEMETRY
+feature OSD
+feature ESC_SENSOR
+
+# serial
+serial 1 2 115200 57600 0 115200
+serial 2 1 19200 57600 0 115200
+serial 3 8192 115200 57600 0 115200
+serial 5 64 115200 57600 0 115200
+serial 7 1024 115200 57600 0 115200
+
+# master
+set gyro_to_use = BOTH
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 2
+set blackbox_device = SDCARD
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 100
```

Comment:
Why is this target specific? This value is dependent on the ESC current sensor, not the FC.

Reply:
fixed